### DISCUSSION TITLE
Setup & Wiring

### DISCUSSION THREAD

**Discussion by DrJon85** [klaasnicolaas/home-assistant-glow#112] [Q&A] [answered]
Hi, i am a complete newbie to all of this and this is one of my first projects of this kind. I am having difficulty in setting up the  wiring for this project, in particular the correct wires to the correct prong of the LED. Can anyone help me with regards to a step by step process to get this completed and integrated with HA.
Many thanks

**Comment by klaasnicolaas** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/20448157/149622836-9efd373f-e45a-4b60-b1df-9f5c33753d3c.png)

There are 2 types of RGB leds and in the [diagrams](https://github.com/klaasnicolaas/home-assistant-glow#led) you will find which pin should be connected to which pin.

**Reply by mpalmergbr** [reply]
A 3volt coin cell across the pins helps to identify the polarity.
